### PR TITLE
added month and year for confused time travelers

### PR DESCRIPTION
**Pull request by samuel-ping**

### CODE REVIEW

File: challenge/src/components/SomeInformationForTimeTravelers.jsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+
+export default class SomeInformationForTimeTravelers extends React.Component {
+    render() {
+        var date = new Date();
+        var monthNum = date.getMonth();
+        var month;
+        // this is gonna be a lotta if statements rip
+        if(monthNum===1) {
```
Comment (mjrb) [inline challenge/src/components/SomeInformationForTimeTravelers.jsx]: pls, consider some way of refactoring this

File: challenge/src/components/SomeInformationForTimeTravelers.jsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+
+export default class SomeInformationForTimeTravelers extends React.Component {
+    render() {
+        var date = new Date();
+        var monthNum = date.getMonth();
```
Comment (mjrb) [inline challenge/src/components/SomeInformationForTimeTravelers.jsx]: _

**Review by mjrb — CHANGES_REQUESTED** [review]

File: challenge/src/components/SomeInformationForTimeTravelers.jsx (hunk 1 above)
Comment (samuel-ping) [reply challenge/src/components/SomeInformationForTimeTravelers.jsx]: i feel attacked but also yes i agree

my if statements also feel attacked

File: challenge/src/components/Pages/Home.jsx [hunk 3]
```diff
@@ -5,6 +5,7 @@ import strings from "../../strings.json";
 import Fade from "react-reveal/Fade";
 import Table from '../Table'
 import GIF from "../Gif"
+import CurrentMonthAndYear from '../SomeInformationForTimeTravelers';
```
Comment (mjrb) [inline challenge/src/components/Pages/Home.jsx]: the convention usually is the component should probably be the same name as the file, but it is also named something else in the other file. make it so everything has the same name `CurrentMonthAndYear`

File: challenge/src/components/SomeInformationForTimeTravelers.jsx [hunk 4]
```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+
+export default class SomeInformationForTimeTravelers extends React.Component {
```
Comment (mjrb) [inline challenge/src/components/SomeInformationForTimeTravelers.jsx]: hilariously js is stupid and its you do this

File: challenge/src/components/SomeInformationForTimeTravelers.jsx [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+            <h1 style={{textAlign: "center"}}>It is {month} {year}</h1>
+        )
+    }
+}
+
+
+
+
+
+// Leaving this here in memory of my poor if statements that got attacked
```
Comment (mjrb) [inline challenge/src/components/SomeInformationForTimeTravelers.jsx]: its good to remember growth, but if you don't remove this, it will haunt you forever

**Review by mjrb — CHANGES_REQUESTED** [review]

File: challenge/src/components/SomeInformationForTimeTravelers.jsx (hunk 5 above)
Comment (samuel-ping) [reply challenge/src/components/SomeInformationForTimeTravelers.jsx]: yes i removed it :(

File: challenge/src/components/Pages/Home.jsx (hunk 3 above)
Comment (samuel-ping) [reply challenge/src/components/Pages/Home.jsx]: done

File: challenge/src/components/SomeInformationForTimeTravelers.jsx (hunk 4 above)
Comment (samuel-ping) [reply challenge/src/components/SomeInformationForTimeTravelers.jsx]: hauh i dont get it

Comment (mjrb) [reply challenge/src/components/SomeInformationForTimeTravelers.jsx]: the component is still named wrong in this file, but it compiles. fun excersize not required for this pr, but find out why and ping me.
please also fix the name here

**Review by mjrb — CHANGES_REQUESTED** [review]

File: challenge/src/components/SomeInformationForTimeTravelers.jsx (hunk 4 above)
Comment (samuel-ping) [reply challenge/src/components/SomeInformationForTimeTravelers.jsx]: excersize

**Review by mjrb — APPROVED** [review]
badda ping
(get it, instead of badda bing)

**Pull request merged**